prow/github/client.go

```diff
@@ -146,9 +147,33 @@ func (t *throttler) Wait() {
 	}
 }
 
+func (t *throttler) Refund() {
+	t.lock.RLock()
+	defer t.lock.RUnlock()
+	select {
```

Comment:
Is this to allow waiting when we're already saturated? Do we care to refund if we're saturated?

Reply:
No this is to *prevent* waiting if we're already saturated since we don't care about refunding in that scenario.